dotbot/controller.py

```diff
@@ -95,28 +96,30 @@ class Controller:
     def __init__(self, settings: ControllerSettings):
         self.dotbots: Dict[str, DotBotModel] = {}
         # self.dotbots: Dict[str, DotBotModel] = {
-        #     "0000000000000001": DotBotModel(
-        #         address="0000000000000001",
-        #         last_seen=time.time(),
-        #         lh2_position=DotBotLH2Position(x=0.5, y=0.5, z=0),
-        #         rgb_led=DotBotRgbLedCommandModel(red=255, green=0, blue=0),
-        #     ),
-        #     "0000000000000002": DotBotModel(
-        #         address="0000000000000002",
-        #         last_seen=time.time(),
-        #         lh2_position=DotBotLH2Position(x=0.2, y=0.2, z=0),
-        #         rgb_led=DotBotRgbLedCommandModel(red=0, green=255, blue=0),
-        #     ),
-        #     "0000000000000003": DotBotModel(
-        #         address="0000000000000003",
-        #         last_seen=time.time(),
-        #     ),
-        #     "0000000000000004": DotBotModel(
-        #         address="0000000000000004",
-        #         application=ApplicationType.SailBot,
-        #         last_seen=time.time(),
-        #         gps_position=DotBotGPSPosition(latitude=48.832313766146896, longitude=2.4126897594949184),
-        #     ),
+            # "0000000000000001": DotBotModel(
+            #     address="0000000000000001",
+            #     last_seen=time.time(),
+            #     lh2_position=DotBotLH2Position(x=0.5, y=0.5, z=0),
+            #     rgb_led=DotBotRgbLedCommandModel(red=255, green=0, blue=0),
+            # ),
+            # "0000000000000002": DotBotModel(
+            #     address="0000000000000002",
+            #     last_seen=time.time(),
+            #     lh2_position=DotBotLH2Position(x=0.2, y=0.2, z=0),
+            #     rgb_led=DotBotRgbLedCommandModel(red=0, green=255, blue=0),
+            # ),
+            # "0000000000000003": DotBotModel(
+            #     address="0000000000000003",
+            #     last_seen=time.time(),
+            # ),
+            # "0000000000000004": DotBotModel(
+            #     address="0000000000000004",
+            #     application=ApplicationType.SailBot,
+            #     last_seen=time.time(),
+            #     wind_angle=135,
+            #     rotation=49,
+            #     gps_position=DotBotGPSPosition(latitude=48.832313766146896, longitude=2.4126897594949184),
+            # ),
```

Comment:
This is mainly unrelated changes, except the wind_angle and rotation of the last one. Can you keep is as it was before but just add the 2 new lines